app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -292,3 +292,9 @@ div.form-group .pficon-info{
 a.btn-info span{
   padding-bottom: 4px;
 }
+
+.blank-slate-pf {
+  font-size: 1.4em;
+  width: 70%;
```

Comment:
Look like 750px works better, 70% can be too much for wide screens.
But please also add a `max-width:90%` as well to make sure it is still readable even on mobile.